src/02-data-types.js

```diff
@@ -127,7 +151,13 @@ function isOdd(num) {}
  *  isTruthy(null);
  *  //> false
  */
-function isTruthy(val) {}
+function isTruthy(val) {
+  if (val){
```

Comment:
remember we can use the `!!` operator to check a value's inherent truthiness. So here we could just return `!!val`.